src/app/address/controllers/summary/confirm.test.js

```diff
@@ -29,6 +25,11 @@ describe("Address confirmation controller", () => {
     req.session.tokenId = sessionId;
     req.session.authParams = {};
 
+    req.headers = {
```

Comment:
I quite like the pattern that @nickhealGDS used where he put the headers in the test setup file so that we don't have to repeat them https://github.com/govuk-one-login/ipv-cri-bav-front/pull/215/files#diff-2ab4bdf8f4f10d730f0f4a63709c981ad4f92031fe9dc587337d6f7d3d5f3863 - what do you think?

Reply:
Good point! In fact, repo had already a mocha helper file. So I just updated it and remove req.headers from test files.